fintech-examples/fintech-ui/src/app/bank/payment/payment-initiate/initiate.component.ts

```diff
@@ -44,7 +44,8 @@ export class InitiateComponent implements OnInit {
   }
 
   onConfirm(): void {
-    let okurl = window.location.pathname;
+    let okurl = this.router.url;
```

Comment:
window.location.pathname is used at least in AisService too. Maybe it makes sense to change it everywhere with next issue.

Reply:
ok